packages/design-tokens/src/tokens/functional/components/prose/colors.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  brand: {
+    Prose: {
+      imageUrl: {
+        default: {
+          value: `url("data:image/svg+xml;charset=UTF-8,%3csvg xmlns='http://www.w3.org/2000/svg' aria-hidden='true' focusable='false' role='img' viewBox='0 -3 16 16' width='16' height='16' fill='black' style='display: inline-block; vertical-align: text-bottom; overflow: visible;'%3e%3cpath d='M2 7.75A.75.75 0 0 1 2.75 7h10a.75.75 0 0 1 0 1.5h-10A.75.75 0 0 1 2 7.75Z'%3e%3c/path%3e%3c/svg%3e ")`,
```

Comment:
**TODO**: This should be gray-9, not black.

Reply:
update: I'm not sure if this will be possible with the current implementation as interpolating values into `url` calls is impossible. Hex codes don't work either.